contracts/script/deploy-localnet.sh

```diff
@@ -0,0 +1,2 @@
+source ../.env.example
+forge script Deploy.s.sol --rpc-url 127.0.0.1:8545 --private-key $DEPLOYER_PRIVATE_KEY --broadcast
```

Comment:
run anvil like so: `anvil --fork-url https://base-mainnet.g.alchemy.com/v2/{your_key}` to fork base mainnet